packages/fast-form-generator-react/src/form/form.utilities.ts

```diff
@@ -1,5 +1,14 @@
 import { clone, cloneDeep, get, isEqual, mergeWith } from "lodash-es";
-import { IBreadcrumbItemConfig, IBreadcrumbItemsConfig, IComponentItem, IFormProps, IFormState, LocationOnChange } from "./form.props";
+import * as tv4 from "tv4";
+import { IBreadcrumbItemConfig, IBreadcrumbItemsConfig, IComponentItem, IFormProps, IFormState } from "./form.props";
+
+export interface ISchemaLocationStringsFromDataLocationStringConfig {
```

Comment:
I'm not really following the purpose of this interface. Did you add it as a product of Code Climate yelling about too many arguments?